scripts/deploy.sh

```diff
@@ -36,7 +36,12 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
   if [[ ! -d /etc/openstack_deploy/ ]]; then
     ./scripts/bootstrap-aio.sh
     pushd ${RPCD_DIR}
-        for i in $(find etc/openstack_deploy/ -type f -iname '*.yml'); do ../scripts/update-yaml.py /$i $i; done
+      for filename in $(find etc/openstack_deploy/ -type f -iname '*.yml');
+        if [[ ! -a /"${filename}" ]]; then
+          cp "${filename}" /"${filename}";
```

Comment:
First off, thanks for using "${...}" notation.  It's just a bit more robust that way.

Not that this is very important in this instance, but the "/" should be inside the quotes, not outside.  In this case it won't make any difference (unless ${filename} starts with a space) but going forward, I think the preferred syntax would be:

cp "${filename}" "/${filename}"